Assets/HoloToolkit/Build/Editor/HoloToolkitCommands.cs

```diff
@@ -1,4 +1,9 @@
-﻿using HoloToolkit.Unity;
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm no lawyer either, but Microsoft has a pretty good moonlighting program and I didn't write this particular code while at work, but at home while developing my own stuff. Not sure about the copyright either way. Mostly I'm concerned with spamming the copyright around as this community grows and outside contributions increase.